detox/test/e2e/06.device.test.js

```diff
@@ -54,4 +54,41 @@ describe('Device', () => {
     await device.shake();
     await expect(element(by.text('Shaken, not stirred'))).toBeVisible();
   });
+
+  describe('device orientation', () => {
```

Comment:
We moved orientation tests outside this file, you can delete this part

Reply:
done